nix-mode.el

```diff
@@ -117,7 +117,7 @@ Valid functions for this are:
 
 (defconst nix-re-quotes "''\\|\"")
 
-(defconst nix-re-comments "#\\|/\\*\\|\\*/")
+(defconst nix-re-comments "#\\|/*\\|*/")
```

Comment:
Hm... Perhaps I don't know emacs regex as well as I thought. But shouldn't the ```*``` be escaped here? You want it to match ```/*``` and ```*/```.

Reply:
Well, my elisp regex is bad as well. I just evalled the older revision of the file and grabbed the value from there. So the one I changed it to here is the one that been in use for ages.